help/view-channel-information.md

```diff
@@ -0,0 +1,51 @@
+# View channel information
+
+When viewing a [topic](/help/introduction-to-topics) or a [channel
+feed](/help/channel-feed), you can find the name, description and subscriber
+count for the current channel directly in your message view.
```

Comment:
I think it might also work when you're looking at any search view withing the channel.